package-res/resources/web/dataapi/models-svc.js

```diff
@@ -54,8 +54,20 @@ pentaho.pda.SvcHandler.prototype.getSources = function(callback, options) {
             var url = this.SERVICE_URL+'?action=listmodels';
             var contextName = options['context'];
             var query = "domainName=&"+((contextName)? 'context='+contextName : "");
-            var jsonStr = pentahoPost( url, query, null, 'text/text' );
-            var rawModels = eval( '('+jsonStr+')' );
+
+            var rawModels;
+            request.post(url, {
```

Comment:
@rfellows, I wish I knew the answer :smile: 
That is how is was implemented, I'm surprised with POST too